pkgs/top-level/all-packages.nix

```diff
@@ -12909,6 +12909,8 @@ let
       pysqlite = pkgs.pythonPackages.sqlite3;
   };
 
+  sooperlooper = callPackage ../applications/audio/sooperlooper { };
```

Comment:
Entries in all-packages.nix should be in alphabetical order and in the appropriate group. You can see the groups by searching for "###" perhaps Applications would be appropriate?